Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -97,10 +98,28 @@ def load_other_tap_strategies(formulae_and_casks_to_check)
     }
     def run_checks(
       formulae_and_casks_to_check,
-      full_name: false, json: false, newer_only: false, debug: false, quiet: false, verbose: false
+      full_name: false, handle_name_conflict: false, json: false, newer_only: false,
+      debug: false, quiet: false, verbose: false
     )
       load_other_tap_strategies(formulae_and_casks_to_check)
 
+      ambiguous_casks = []
+      if handle_name_conflict
+        ambiguous_casks = formulae_and_casks_to_check
+                          .group_by { |item| formula_or_cask_name(item, full_name: true) }
+                          .select { |_name, items| items.length > 1 }
+                          .values.flatten
```

Comment:
```suggestion
                          .values
                          .flatten
```

but, given you don't use the values for the `.select` you could use `.values` before that.